packages/@ourworldindata/grapher/src/captionedChart/CaptionedChart.tsx

```diff
@@ -137,64 +155,61 @@ export class CaptionedChart extends React.Component<CaptionedChartProps> {
         )
     }
 
-    @computed protected get chartHeight(): number {
-        const controlsRowHeight = this.controls.length ? CONTROLS_ROW_HEIGHT : 0
-        return Math.floor(
-            this.bounds.height -
-                this.header.height -
-                controlsRowHeight -
-                this.footer.height -
-                PADDING_ABOVE_FOOTER
-        )
-    }
-
-    // todo: should we remove this and not make a distinction between map and chart tabs?
-    @computed protected get isMapTab(): boolean {
-        return this.manager.tab === GrapherTabOption.map
-    }
-
     @computed protected get bounds(): Bounds {
         return this.props.bounds ?? this.manager.tabBounds ?? DEFAULT_BOUNDS
     }
 
-    // The bounds for the middle chart part
-    @computed protected get boundsForChart(): Bounds {
-        const topPadding = this.isMapTab
-            ? 0
-            : this.manager.type === ChartTypeName.Marimekko
-            ? PADDING_BELOW_HEADER / 2
-            : PADDING_BELOW_HEADER
-        return new Bounds(0, 0, this.bounds.width, this.chartHeight)
-            .padWidth(OUTSIDE_PADDING)
-            .padTop(topPadding)
-            .padBottom(OUTSIDE_PADDING)
+    @computed protected get boundsForChartArea(): Bounds {
+        const { bounds, chartHeight } = this
+        return new Bounds(0, 0, bounds.width, chartHeight).padWidth(
+            FRAME_PADDING
+        )
```

Comment:
Note for the reviewers: The vertical padding (`padTop`, `padBottom`) has been removed from the bounds of the chart area. As a result, some chart types (e.g. Scatter plot) would sometimes overflow. To counteract, padding has been applied to the axis of specific chart types when necessary (see `ScatterPlotChart.tsx`, for example).